.goreleaser.yml

```diff
@@ -0,0 +1,83 @@
+before:
+  hooks:
+    - go mod download
+
+env:
+  - GO111MODULE=on
+
+builds:
+  -
+    id: tt
+    binary: tt
+    dir: cli
+
+    env:
+      - CGO_ENABLED=0
+
+    flags:
+      - -tags=go_tarantool_ssl_disable
+
+    ldflags:
+      - -s -w
+      - -X github.com/tarantool/tt/cli/version.gitTag={{ .Tag }}
+      - -X github.com/tarantool/tt/cli/version.gitCommit={{ .ShortCommit }}
+
+    goos:
+      - linux
+    goarch:
+      - amd64
+
+    hooks:
+      pre:
+        - mage PatchCC
+        - mage GenerateGoCode
+
+archives:
+  -
+    id: "tt"
+    builds: ['tt']
+    format: tar.gz
+    name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}"
+    files:
+      - README.rst
+      - LICENSE
+      - CHANGELOG.md
+
+snapshot:
+  name_template: "{{ .Tag }}-{{ .ShortCommit }}"
+
+changelog:
+  skip: true
+
+release:
+  draft: true
+
+nfpms:
+  -
+    id: "tt"
+    builds: ['tt']
+    formats:
+        - deb
+        - rpm
+    homepage: "https://github.com/tarantool/tt"
+    maintainer: "Pavel Balaev <balaev@tarantool.org>"
```

Comment:
https://github.com/tarantool/ecosystem-team#gatekeepers (yet Leonid Vasiliev :) )

Reply:
Fixed. AFAIK this is a package maintainer record.